pkg/pub_integration/lib/src/pub_puppeteer_helpers.dart

```diff
@@ -187,6 +192,39 @@ extension PubPageExt on Page {
     await _waitForModelHidden();
   }
 
+  /// Waits until the given selectors are visible and their UI top-left position
```

Comment:
Sounds completely crazy that we have to make a hack like this. Should this not be built into the framework somehow?